test/rules/avoid_js_rounded_ints.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N avoid_js_rounded_ints`
+
+final i = 1; // OK
+final min = -9007199254740991; // OK
+final max = 9007199254740991; // OK
+
+final minErr = -9007199254740993; // LINT
+final maxErr = 9007199254740993; // LINT
+
+final notRounded = 1000000000000000000; // OK
+final rounded = 1000000000000000001; // LINT
```

Comment:
Also test with -9223372036854775808
value.abs() for this number is negative on the 64-bit integer VM.
Lucky it is not rounded! (-2^63), so maybe comment that the code works for this edge case.

Reply:
done

@bwilkerson `9223372036854775808` has `node.value == null`. Is it a bug ?

Reply:
It sounds like a bug. Every integer literal that is a valid 64-bit integer ought to have a non-null value.